bool isFirstBoot
float32 degree
float32 distance
---
bool isSuccess